### PR TITLE
Re-enable bitmanip tests

### PR DESCRIPTION
**Pull request by rswarbrick**
Also tweak the `verification.rst` documentation a little bit to reflect how things now work (ovpsim won't).

### CODE REVIEW

File: doc/03_reference/verification.rst [hunk 1]
```diff
@@ -125,14 +122,9 @@ to tell the RISCV-DV code where to find them:
     export RISCV_GCC="$RISCV_TOOLCHAIN/bin/riscv32-unknown-elf-gcc"
     export RISCV_OBJCOPY="$RISCV_TOOLCHAIN/bin/riscv32-unknown-elf-objcopy"
     export SPIKE_PATH=/path/to/spike/bin
-    export OVPSIM_PATH=/path/to/ovpsim/bin
-
-(Obviously, you only need to set ``SPIKE_PATH`` or ``OVPSIM_PATH`` if
-you have installed the corresponding instruction set simulator)
 
 .. _Spike: https://github.com/riscv/riscv-isa-sim
```
Comment (GregAC) [inline doc/03_reference/verification.rst:126]: Do we want to keep this here? Given you need to use LRSpike best to remove it to avoid confusion

**Comment by GregAC** [comment]
Regression run here: https://dev.azure.com/lowrisc/lowrisc-private/_build/results?buildId=68733&view=results

**Comment by GregAC** [comment]
Well all bitmanip tests failed, but that's with cosim enabled.

Cosim is meant to be disabled for those specific tests but I guess the disable function has broken somehow!

Rerunning the regression without cosim but I'm happy to merge this as those test failures don't cause any big issues (and it may all pass without cosim enabled).

**Comment by vogelpi** [comment]
Thanks @rswarbrick , this looks great!

**Pull request merged**